.gitignore

```diff
@@ -1,11 +1,94 @@
-target
-bin
-.classpath
-.settings
-.project
+### Linux
```

Comment:
I don't think, that all file types are need. 
Do You generated this file automaticly?